packages/svelte/src/compiler/types/template.d.ts

```diff
@@ -438,6 +438,9 @@ export namespace AST {
 		expression: Identifier;
 		parameters: Pattern[];
 		body: Fragment;
+		metadata: {
```

Comment:
So that it doesn't show up in the generated public types

```suggestion
		/** @internal */
		metadata: {
```